### PR TITLE
qt4-x11: removed as qt4-mac works just fine

### PR DESCRIPTION
**Pull request by yan12125**
#### Description

Ref: https://trac.macports.org/ticket/20309#comment:43

Apparently nobody cares about this port, and it blocks migrating openssl to 1.1.x (#3822).

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
Not tested

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by yan12125** [comment]
@macportsbot retry

**Comment by cjones051073** [comment]
Obviously OK for root5, as you are just removing stuff already commented out ;)

**Comment by michaelld** [comment]
I think this is a fine idea! One less port to have to (try to) maintain ...

File: security/Vidalia/Portfile [hunk 1]
```diff
@@ -25,26 +26,12 @@ depends_lib-append  port:tor
 
 patchfiles          patch-TorSettings.h.diff
 
+configure.args ${qt_cmake_defines}
```
[the remark this answers is not in the document]
Comment (yan12125) [reply security/Vidalia/Portfile]: Thanks for pointing it out! Fixed.

**Comment by yan12125** [comment]
Added references to remaining Trac tickets for qt4-x11 to the commit message of the first commit.

**Comment by cjones051073** [comment]
So, are we ready to hit go on this ?

**Comment by michaelld** [comment]
+1 from me to merge

**Comment by yan12125** [comment]
Thank you both for reminding. I think 9 days is long enough. Let's move on!

**Pull request merged**